tests/pytests/test_search_params.py

```diff
@@ -325,15 +327,69 @@ def test_numeric_range(env):
     env.assertEqual(res2, res1)
 
     res1 = env.cmd('FT.SEARCH', 'idx', '@numval:[(102 +inf]', 'NOCONTENT')
-    env.assertEqual(res1, [3, 'key3', 'key4', 'key5'])
+    env.assertEqual(res1, [4, 'key3', 'key4', 'key5', 'key7inf'])
     res2 = env.cmd('FT.SEARCH', 'idx', '@numval:[($min $max]', 'NOCONTENT', 'PARAMS', '4', 'min', '102', 'max', '+inf')
     env.assertEqual(res2, res1)
 
     res1 = env.cmd('FT.SEARCH', 'idx', '@numval:[-inf, (105]', 'NOCONTENT')
-    env.assertEqual(res1, [4, 'key1', 'key2', 'key3', 'key4'])
+    env.assertEqual(res1, [5, 'key1', 'key2', 'key3', 'key4', 'key6neg'])
     res2 = env.cmd('FT.SEARCH', 'idx', '@numval:[$min ($max]', 'NOCONTENT', 'PARAMS', '4', 'min', '-inf', 'max', '105')
     env.assertEqual(res2, res1)
 
+    res1 = env.cmd('FT.SEARCH', 'idx', '@numval:[-101 101]', 'NOCONTENT')
```

Comment:
Add comment on the block? (parameters with sign and/or exclusive ranges)

Reply:
OK. Done.